code/src/index.css

```diff
@@ -1,13 +1,339 @@
+  width: 250px;
+  padding: 10px;
+  margin-top: 10px;
+  background:#fbf0f0;
+  font-size: 15px;
+  border: none;
+  border-radius: 5px;
+  font-family: 'Roboto Mono', monospace;
+}
+/* Checkbox styling */
+
+.checkbox {
+  background-color: #E3CECE;
+  height: 100vh;
+  padding: 150px 15px;
+  display: flex;
+  flex-direction: column;
+}
+
+.checkbox-group{
+  display: flex;
+  flex-direction: column;
+}
+
+.checkbox-container{
+  display:block;
+  position: relative;
+  padding-left: 25px;
+  margin: 10px 5px;
+  font-size: 15px;
+  user-select: none; 
+  font-family: 'Roboto Mono', monospace;
+}
+
+.checkbox-container input{
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 0;
+  width: 0;
+}
+
+.custom-checkbox{
+  position: absolute;
+  height: 19px;
+  width: 19px;
+  background-color: #eee;
+  border-radius: 4px;
+  cursor: pointer; 
+}
+
+.custom-checkbox{
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 19px;
+  width: 19px;
+  background-color: #eee;
+  border-radius: 4px;
+  cursor: pointer; 
+}
+
+.checkbox-container:hover input ~ .custom-checkbox{
+  background-color: #ccc;
+}
+
+.checkbox-container input:checked ~ .custom-checkbox{
+  background-color: #444444;
+}
+
+.custom-checkbox:after{
+  content: "";
+  position: absolute;
+  width: 3px;
+  height: 6px;
+  border: solid white;
+  border-width: 0 3px 3px 0;
+  top: 5px;
+  left: 7px; 
+  transform: rotate(45deg);
+  display: none;
+}
+
+.checkbox-container input:checked ~ .custom-checkbox:after{
+  display: block;
+}
+
+/* Radiobutton styling */
+
+.radiobutton {
+  background-color: #E6D2D2;
+  height: 100vh;
+  padding: 150px 15px;
+  display: flex;
+  flex-direction: column;
+  font-size: 15px;
+  font-family: 'Roboto Mono', monospace;
+}
+
+.age-group{
+  display: flex;
+  flex-direction: column;
+}
+
+.radiobuttoncontainer{
+    display: block;
+    position: relative;
+    padding-left: 25px;
+    margin-top: 10px;
+    margin: 10px 5px;
+    font-size: 16px;  
+    -webkit-user-select: auto;
+}
+
+.radiobuttoncontainer input{
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 0;
+  width: 0;
+}
+
+.custom-radio{
+  position: absolute; 
+  top: 0;
+  left: 0;
+  height: 19px;
+  width: 19px;
+  background: #eee;
+  cursor: pointer;
+  border-radius: 50%;
+  Font-Family: 'Cabin', Sans-Serif;
+  Font-Size: 15px;
+}
+
+.radiobuttoncontainer:hover input ~ .custom-radio{
+  background-color: #ccc;
+} 
+
+.radiobuttoncontainer input:checked ~ .custom-radio{
+  background-color: #444444; 
+}
+
+.custom-radio:after {
+  content: "";
+  position: absolute;
+  top: 6px;
+  left: 6px;
+  width: 7px;
+  height: 7px;
+  background-color: #fff;
+  border-radius: 50%; 
+  display: none;
+}
+
+.radiobuttoncontainer input:checked ~ .custom-radio:after{
+  display: block; 
+}
+
+/* Something else */ 
+
+.summary{
+  height: 100vh;
+  background: #FAE3E3;
+  padding: 150px 15px;
+  font-family: 'Roboto Mono', monospace;
+}
```

Comment:
Would be nice if this was not so wide: 
<img width="903" alt="Skärmavbild 2020-11-06 kl  10 34 47" src="https://user-images.githubusercontent.com/51529201/98350483-b2bf0100-201b-11eb-82b0-1f95ba144eab.png">